sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -355,9 +361,9 @@ def _run_bundle_multiple_times_for_testing(
       self,
       runner_execution_context,  # type: execution.FnApiRunnerExecutionContext
       bundle_manager,  # type: BundleManager
-      data_input,
+      data_input,  # type: Dict[str, PartitionableBuffer]
       data_output,  # type: DataOutput
-      fired_timers,
+      fired_timers,  # type: Mapping[Tuple[str, str], execution.PartitionableBuffer]
```

Comment:
note to self:  standardize on `PartitionableBuffer` vs `execution.PartitionableBuffer`

Reply:
resolved